caramel/models.py

```diff
@@ -42,14 +30,15 @@ def _crypto_patch():
     https://github.com/pyca/pyopenssl/pull/115 has a pull&fix for it
     https://github.com/pyca/pyopenssl/issues/129 is an open issue
     about it."""
-    _crypto._lib.ASN1_STRING_set_default_mask_asc(b'utf8only')
+    _crypto._lib.ASN1_STRING_set_default_mask_asc(b"utf8only")
```

Comment:
These changes are  conceptually good, (looks like black)  but have the wrong commit message.

Inject a change before moving the database code into the tree where it only runs "black" and removes the old python2.6 compat. give it a clear commit message that it's an automated change.
"Make lints happy" or similar.